StandardLibrary/src/Time.dfy

```diff
@@ -36,11 +36,19 @@ module {:extern "Time"} Time {
     // be negative.
     ensures res >= 0
 
+  // Returns the number of milliseconds of CPU used by the current process so far.
+  // Time is represented as signed over unsigned because java does not do unsigned
+  // values - net can do both so we are able to change the representation to signed.
+  method {:extern "GetCurrentCPU"} GetCurrentCPU() returns (res: int64)
```

Comment:
I'm not a big fan of this name -- maybe `GetProcessCpuTimeMillis`?

Reply:
Done